### ISSUE TITLE
Linting: PMC Syntax improvement

### ISSUE THREAD

**Issue by reflodark** [iSorp/macro-executor#32]
The following syntax is valid:
```
IF [R[_led_optional_stop + pmc_offset].2] THEN
    RETURN
ENDIF

#v01 =  R[_led_optional_stop + pmc_offset].2
#v02 = R[_led_optional_stop + pmc_offset].[BIT_2]
```

whereas:
`#v03 = R[_led_optional_stop + pmc_offset].BIT_2`

is invalid.

I will try to make a pullrequest myself, just don't expect it any time soon ;-)

**Comment by reflodark** [comment]
Solved with #38

**Comment by iSorp** [comment]
Not solved by #38
